src/c++/perf_analyzer/request_rate_manager.cc

```diff
@@ -263,7 +263,7 @@ RequestRateManager::Infer(
 
     thread_config->is_paused_ = false;
 
-    uint32_t seq_id = 0;
+    uint32_t seq_stat_id = 0;
```

Comment:
Any particular reason why you're adding `stat` to the variable name? It makes it seem conceptually different from the "sequence ID" concept, which is well defined here:

https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_sequence.md

Reply:
I'm going to change it even further to seq_stat_index.  This is the index into the sequence_stat vector, which is very different than the sequence ID that is passed to triton. The fact that these two concepts were using the same variable name was very confusing to me.